rkt/run.go

```diff
@@ -445,7 +445,9 @@ func (e *envFileMap) Set(s string) error {
 
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
-		// TODO parse '\' to skip new line character
+		if len(scanner.Text()) == 0 {
+			continue
```

Comment:
@alban `len(s) == 0` is wrong but I think it should be `len(s) < 2` because if a line do not have an `=` `SplitN()` returns one element. Do we want to allow that? I prefer that if the user wants an empty variable have to write `variable=`?